fares.md

```diff
@@ -23,7 +23,7 @@ on `fare_class`, and the cost of the fare is returned by the price field.
 
 *[`routes_ft.txt`](/files/routes_ft.md)*
 
-`route_id`	| `mode` 		| `fare_class` 	| `proof_of_payment`										
+`route_id`	| `mode` 		| `fare_period` 	| `proof_of_payment`										
```

Comment:
Lisa, can you make these little changes so I don't have to go through the whole process. It just seems to be a bit much for editing text documents. Thanks!